src/content-scripts/gemini-tracker/gemini-history-button-controller.js

```
@@ -0,0 +1,1137 @@
+          const sendButtonSelectors = [
+            'button:has(mat-icon[data-mat-icon-name="send"])',
+            "button.send-button",
+            'button[aria-label*="Send"]',
+            'button[data-test-id="send-button"]',
+          ];
+
+          const sendButtons = document.querySelectorAll(sendButtonSelectors.join(", "));
+
+          // Check if there are any enabled buttons with text in the textarea
+          const textarea = document.querySelector("textarea");
+          if (textarea && textarea.value.trim() && sendButtons.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)